tailwind.config.js

```diff
@@ -96,14 +102,88 @@ module.exports = {
       },
       white: theme("colors.background")
     }),
-    transitionDuration: {
-      75: "75ms",
-      150: "150ms",
-      300: "300ms",
-      500: "500ms"
+    backgroundColor: theme => theme("colors"),
+    animation: {
+      "in": "in 300ms ease-in-out",
+      "in-down": "in-down 300ms ease-in-out",
+      "in-up": "in-up 300ms ease-in-out",
+      "in-scale": "in-scale 300ms linear"
     },
-    transitionTimingFunction: "cubic-bezier(0.215, 0.440, 0.420, 0.880)"
+    keyframes: {
+      "in": {
+        "0%": {
+          opacity: 0
+        },
+        "100%": {
+          opacity: 1
+        }
+      },
+      "in-down": {
+        "0%": {
+          opacity: 0,
+          transform: "translate3D(0, -5px, 0)"
+        },
+        "100%": {
+          opacity: 1,
+          transform: "translate3D(0, 0, 0)"
+        }
+      },
+      "in-up": {
+        "0%": {
+          opacity: 0,
+          transform: "translate3D(0, 5px, 0)"
+        },
+        "100%": {
+          opacity: 1,
+          transform: "translate3D(0, 0, 0)"
+        }
+      },
+      "in-scale": {
+        "0%": {
+          opacity: 0,
+          transform: "scale3D(0.95, 0.95, 1)"
+        },
+        "100%": {
+          opacity: 1,
+          transform: "scale3D(1, 1, 1)"
+        }
+      }
+    },
+    extend: {
+      transitionProperty: {
+        margin: "margin",
+        color: "color"
+      },
+      transitionTimingFunction: {
+        cubic: "cubic-bezier(0.215, 0.440, 0.420, 0.880)"
+      }
+    }
   },
+  plugins: [
+    plugin(function({ addUtilities }){
+      const newUtilities = {
+        ".word-break": {
+          "word-wrap": "break-word",
+          "word-break": "break-word"
+        },
+        ".focus-style-base": {
+          "outline-offset": 0,
+          "outline-color": "transparent",
+          "transition": "outline-offset 100ms ease-in-out, outline-color 100ms ease-in-out"
+        },
+        ".focus-style-outset": {
+          "outline": "2px solid var(--calcite-ui-blue-1)",
+          "outline-offset": "2px"
+        },
+        ".focus-style-inset": {
```

Comment:
Remove `-style` for the sake of brevity?

Reply:
I like it.
I like it a `breve latte`.